CIP-0071/README.md

```diff
@@ -0,0 +1,246 @@
+---
+CIP: 71
+Title: Non-Fungible Token (NFT) Proxy Voting Standard
+Authors: Thaddeus Diamond <support@wildtangz.com>
+Comments-URI:
+Status: Proposed
+Type: Informational
```

Comment:
I am just curious why you chose to label this as `Informational` (rather than `Process`) because what's described here seems to depend on a fair bit of interoperability... from [latest CIP-0001 draft](https://github.com/cardano-foundation/CIPs/blob/cip-0001-rework/CIP-0001/README.md):

Type          | Description
--- | ---
Process       | A process defines a common standard to follow to achieve a certain degree of interoperability between various actors. It can be a formal document describing a specific procedure or a set of recommendations/guidelines on one particular topic.
Informational | An informational CIP documents a particular design decision. These are often written a posteriori to address a gap in understanding and knowledge sharing.

Reply:
Great question! I honestly did not know about this categorization I was just copying from prior CIPs.  Will update to be "Process".